core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -709,7 +711,7 @@ void load()
    *
    * @since 2.0.0
    */
-  default ImportSourceArguments addFilesTo(String tableName) {
+  default ImportDestinationArguments importDirectory(String directory) {
```

Comment:
Since its a fluent method and that might not be obvious at first, it would be nice to add an example of calling this to the javadoc.

Reply:
This is a pre-existing issue, so it does not need to hold this up.  I think these changes look good.